hostagent.proto

```diff
@@ -64,6 +64,16 @@ message Command {
     message Install {
         string id = 1;
         optional string cloudinit = 2;  // cloudinit is the yaml configuration to optionally pass to an instance.
+        /*
+          Optional url to a WSL rootfs image we will download and import from. It will be registered as "id".
+          If this entry is not set, Ubuntu Pro for WSL will download a package from the Microsoft store matching "id".
```

Comment:
Not a package matching the ID (e.g. `Ubuntu-Preview` is `Ubuntu (Preview)` on the store), but rather matching the `Name` as defined in https://github.com/microsoft/WSL/blob/master/distributions/DistributionInfo.json

Reply:
pushed an updated comment, tell me what you think :)